spacy/cli/_util.py

```diff
@@ -582,6 +582,29 @@ def setup_gpu(use_gpu: int, silent=None) -> None:
             local_msg.info("To switch to GPU 0, use the option: --gpu-id 0")
 
 
+def walk_directory(path: Path, suffix: Optional[str] = None) -> List[Path]:
```

Comment:
This recursively gets all non-hidden files with a certain suffix in a directory? It would be good to have a docstring, as the function name definitely doesn't make that clear to me. 

Additionally I think this can be simplified a lot, something like:

```
paths = path.rglob(f"*/*.{suffix}")
out = []
for path in paths:
    hidden = False
    for part in path.parts:
        if part.startswith("."):
            hidden = True
    if not hidden:
        out.append(path)
return out
```

And also... do we care about hidden files? I'm not sure there's a need for them in data directories, and this isn't how hidden files work on Windows, for example.

Reply:
OK, I see now that this was moved from the `convert` code, in which case my comments don't apply to this PR. But we may want to revisit this.